packages/action-group/test/action-group.test.ts

```diff
@@ -338,9 +331,13 @@ describe('ActionGroup', () => {
             'mouse2: should not be focused on the fourth button'
         ).to.equal(-1);
 
+        await elementUpdated(el);
```

Comment:
waiting for the element to update eliminated the need for` aTimeout` on line 321